ext/TensorAlgebraGradedUnitRangesExt/TensorAlgebraGradedUnitRangesExt.jl

```diff
@@ -1,8 +1,29 @@
 module TensorAlgebraGradedUnitRangesExt
-using GradedUnitRanges: AbstractGradedUnitRange, tensor_product
-using TensorAlgebra: TensorAlgebra
+
+using GradedUnitRanges: AbstractGradedUnitRange, dual, tensor_product
+using GradedUnitRanges.BlockArrays: Block, blocklengths, blocksize
+using Random: AbstractRNG
+using TensorAlgebra: TensorAlgebra, random_unitary
 
 function TensorAlgebra.:⊗(a1::AbstractGradedUnitRange, a2::AbstractGradedUnitRange)
   return tensor_product(a1, a2)
 end
+
+function TensorAlgebra.dual(a::AbstractGradedUnitRange)
+  return dual(a)
+end
```

Comment:
```suggestion
TensorAlgebra.dual(a::AbstractGradedUnitRange) = GradedUnitRanges.dual(a)
```
It might read slightly easier to be explicit with the qualifiers, otherwise it's easy to get confused which one `dual` refers to?

Reply:
Makes sense.